src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Backup/BackupStep.cs

```diff
@@ -37,11 +36,12 @@ public class BackupStep : UpgradeStep
             WellKnownStepIds.NextProjectStepId,
         };
 
-        public BackupStep(UpgradeOptions options, ILogger<BackupStep> logger, IUserInput userInput)
+        public BackupStep(UpgradeOptions options, ILogger<BackupStep> logger, IUserInput userInput, IUpgradeContextProperties properties)
```

Comment:
The properties should still be accessible off the context rather than a service that is injected.

Reply:
This was a complete brain fart. It should be corrected now in the latest push.